### PR TITLE
Bump version v0.16.0-rc.1

### PR DESCRIPTION
**Pull request by elezar**

### CODE REVIEW

File: hack/generate-changelog.sh [hunk 1]
```diff
@@ -77,4 +77,4 @@ echo ""
 echo "### Version $VERSION"
 
 # Iterate over the commit messages and ignore the ones that start with "Merge" or "Bump"
-git log --pretty=format:"%s" $REFERENCE..@ | grep -Ev "(^Merge )|(^Bump)" |  sed 's/^\(.*\)/- \1\\n/g' 
+git log --pretty=format:"%s" $REFERENCE..@ | grep -Ev "(^Merge )|(^Bump)" |  sed 's/^\(.*\)/- \1/g'
```
Comment (cdesiniotis) [inline hack/generate-changelog.sh:80]: This is my first time seeing this. Nice!

**Review by cdesiniotis — APPROVED** [review]

File: hack/generate-changelog.sh (hunk 1 above)
Comment (elezar) [reply hack/generate-changelog.sh:80]: It does generate a very VERBOSE changelog, but it's probably better to add better checks and commit messages than to manually generate those anyway.

**Review by ArangoGutierrez — APPROVED** [review]
:)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,50 @@
 ## Changelog
 
+### Version v1.16.0-rc.1
+
+- Skip container updates if only CDI is selected
```
Comment (elezar) [inline CHANGELOG.md:5]: We definitely need to filter these a bit more strictly.

Comment (ArangoGutierrez) [reply CHANGELOG.md:5]: is just a helper script, to filter them better we would need to be more strict with Commit naming so we can use regex filters, or have an LLM action as a helper for this

**Pull request merged**